client/src/Controllers/getProfileData.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ProfileInfoResponse, request, UserResponse, User, Works, TopTenReviews, ProfileDataProps,
+} from '../utils';
+
+const getUserProfileData = async (
+  {
+    setPage,
+    setIsLoading,
+    successCB,
+    failedCB,
+    id,
+  } : ProfileDataProps,
+) => {
+  try {
+    setPage(1);
+    setIsLoading(true);
+    const response : ProfileInfoResponse = await request('get', `/providers/${id}`);
+    const { data } : {data: UserResponse} = response;
+    const { user } : {user: User} = data;
+    const { works }: { works: Works } = data;
+    const { reviews }: { reviews: TopTenReviews[] } = data;
+    const { totalReviews }: { totalReviews: number} = data;
+    const { count } = data;
+    successCB(user, works, reviews, totalReviews, count);
+    setIsLoading(false);
```

Comment:
```suggestion
    setIsLoading(false);
```
Remove this line immediately when the response come back